lib/matplotlib/patches.py

```diff
@@ -234,8 +234,9 @@ def get_linewidth(self):
 
     def get_linestyle(self):
         """
-        Return the linestyle.  Will be one of ['solid' | 'dashed' |
-        'dashdot' | 'dotted']
+        Return the linestyle.
```

Comment:
Perhaps make the whole thing a single Returns section?

Reply:
Actually the content is incomplete. There can be much more return values. See `set_linestyle()`. For simplicity I don't list them here again but just remove the comment on the returned content.